pages/Ethereum/Identity.md

```diff
@@ -1,48 +1,28 @@
 ---
-name: Overview
-route: /Identity/Overview
-menu: Identity
+name: Identity
+route: /Ethereum/Identity
+menu: "DSNP: Ethereum"
 ---
 
 # Identity
 
-This specification is intended to cover the concept of identity within the protocol and how we represent it.
-
 ## Purpose
 
 1. Provide the interface for a DSNP identity.
 1. Specify the delegation model and related interface.
 1. Specify the ownership model and related interface.
 1. Provide the list of EIPs that must be supported by a DSNP compatible identity.
```

Comment:
Personally I always put the hyphen in compound adjectives like "DSNP-compatible", to make the binding clear and avoid any possible ambiguity.  Obviously, this comment would apply to all occurrences of that phrase in this spec, but I'm only noting it here, so as to contribute rather than annoy.